src/app/page.tsx

```diff
@@ -15,8 +16,9 @@ export default function Home() {
       <section id="listings">
         <CurrentListing />
       </section>
-      <section>services</section>
-      <section>contact info</section>
+      <section>
+        <Footer />
```

Comment:
We can remove this component from this page. It's being piped into the layout.tsx file so it's already live.